app/views/content/steps-to-become-a-teacher/_step_3_consider_getting_experience.html.erb

```diff
@@ -1,3 +1,9 @@
 <p>Experiencing life in a school can help you decide if teaching is right for you and who you want to teach. This is a good way to give you a taste of what the classroom is really like.</p>
 
+<%= render Content::ExpanderComponent.new(
+    title: "experience in a classroom in England",
+    text: "<p>To get experience in a school in England, you'll need the right to work in the UK. If you don't have experience in a classroom, you can still apply for teacher training.</p>
```

Comment:
As per the comments on the GSE page, I think this is a bit misleading - you need right to work if you are using our GSE service but is that true for all voluntary positions mentioned on the GSE page?

I wonder if we'd be better not having this expander on the steps page?

Also I think the title sets the expectation that they can get school experience?

Reply:
remove